src/components/Toast/ErrorToast.tsx

```diff
@@ -20,10 +20,10 @@ export const ErrorToast: FC<TransactionToastComponentProps> = ({
           <FortIconWarning className="col-span-full row-span-full h-5 w-5 fill-red-500" />
         </div>
         <div className="ml-3">
-          <span className="mb-1 text-sm">{message}</span>
+          <span className="text-md mb-1 font-semibold">{message}</span>
```

Comment:
`text-md` isn't a class -- no class needed if you want the base font size. Same issue w/ all three toast components.

Reply:
right